contracts/src/LightClient.sol

```diff
@@ -362,40 +361,44 @@ contract LightClient is Initializable, OwnableUpgradeable, UUPSUpgradeable {
         uint256 updatesCount = stateHistoryCommitments.length;
 
         // Handling Edge Cases
-        // Edgecase 1: The block is in the future or
-        // before HotShot was live, allow for at least two updates before considering HotShot live
+        // Edgecase 1: blockNumber is greater than current block.number or
+        // less than 3 updates exist which means the history is insufficient to determine a lag.
         if (blockNumber > block.number || updatesCount < 3) {
```

Comment:
it can be but the idea here is that we're giving after the first two updates we can consider the contract stable and thus we expect the updates to continue - i don't think it's necessary to be configurable

Reply:
I was little confused when reading the number 3, we should add more explanation of where does this number come from? does it come from HotShot consensus protocol or just a heuristic?